engine/include/cubos/engine/ui/canvas/plugin.hpp

```diff
@@ -0,0 +1,27 @@
+/// @dir
+/// @brief @ref canvas-plugin plugin directory.
+
+/// @file
+/// @brief Plugin entry point.
+/// @ingroup canvas-plugin
+
+#pragma once
+
+#include <cubos/engine/prelude.hpp>
+
+namespace cubos::engine
+{
+    /// @defgroup canvas-plugin Canvas
+    /// @ingroup engine
+    /// @brief Adds UI canvas and elements.
+
+    CUBOS_ENGINE_API extern Tag elementVPUpdateTag;
+    CUBOS_ENGINE_API extern Tag elementPropagateTag;
+    CUBOS_ENGINE_API extern Tag uiBeginTag;
+    CUBOS_ENGINE_API extern Tag uiDrawTag;
```

Comment:
Document what each tag does and what it's useful for.
Also, if any of those isn't useful outside the plugin, we should omit it from the interface and keep it in the `src/` dir.